code128.py

```diff
@@ -0,0 +1,66 @@
+# taken from https://bitbucket.org/whitie/python-barcode
+
```

Comment:
rgr.  yea we should just paste the MIT license here at the top of this page.  even if they relicense, THIS COPY of the code is still MIT licensed.